merkle/objhasher/objhasher.go

```diff
@@ -16,42 +16,28 @@
 package objhasher
 
 import (
-	"crypto/sha256" // Use SHA256 to match ObjectHash.
-
+	"crypto"
 	"github.com/benlaurie/objecthash/go/objecthash"
+	"github.com/google/trillian/merkle"
+	"github.com/google/trillian/merkle/rfc6962"
 )
 
 // ObjectHasher uses ObjectHash to compute leaf hashes.
-var ObjectHasher = &objhasher{}
+var ObjectHasher merkle.TreeHasher = &objhasher{
+	// Use SHA256 to match ObjectHash.
+	TreeHasher: rfc6962.New(crypto.SHA256),
```

Comment:
This is by design.  The object hasher is exactly the same as the rfc6962 hasher, except that it has a different leaf hash algorithm.